### PR TITLE
Addition of new showcase Priority to fronts tool

### PR DESCRIPTION
**Pull request by tonytw1**
## What's changed?

Adds a new `showcase` Priority to the fronts tool.

Screenshot 2021-08-18 at 14 15 59

The priority will be used to house the fronts which will be the sources for our Google Showcase feeds.

## Implementation notes

Tests ok in CODE.
CODE frontend seems happy to render fronts with this new priority.

Merging this PR makes the Showcase integration general knowledge to all fronts users.

### General
- [ ] 🤖 Relevant tests added
- [ ] ✅ CI checks / tests run locally
- [x] 🔍 Checked on CODE

### Client
- [x] 🚫 No obvious console errors on the client (i.e. React dev mode errors)
- [x] 🎛️ No regressions with existing user interactions (i.e. all existing buttons, inputs etc. work)
- [ ] 📷 Screenshots / GIFs of relevant UI changes included

### CODE REVIEW

**Review by juanvaccari — APPROVED** [review]

**Review by davidfurey — COMMENTED** [review]
lgtm but it would be good to get a review from someone on the ed tools team before merging

**Review by jonathonherbert — APPROVED** [review]
Code looks fine, can see the new priority locally 👍

**Pull request merged**
